manifests/init.pp

```diff
@@ -284,84 +284,119 @@
       $package_name_default      = 'nscd'
       $package_source_default    = undef
       $service_name_default      = 'nscd'
-      case $::operatingsystemmajrelease {
-        '5': {
```

Comment:
Could you just change the case for EL7 to be `'7', '2015', '2016':`   and adding a comment that the 2015 and 2016 are for Amazon?